code/week02/src/Week02/Homework1.hs

```diff
@@ -32,23 +32,28 @@ import           Text.Printf          (printf)
 {-# INLINABLE mkValidator #-}
 -- This should validate if and only if the two Booleans in the redeemer are equal!
 mkValidator :: () -> (Bool, Bool) -> ValidatorCtx -> Bool
-mkValidator _ _ _ = True -- FIX ME!
+mkValidator () (b1, b2) _ = traceIfFalse "wrong redeemer" $ b1 == b2
```

Comment:
Someone tried to introduce their own type as a tuple here: https://stackoverflow.com/questions/67164090/typed-version-of-plutus-pionneers-homework01-week02

Not sure why it does not work